### PR TITLE
completed react-stopwatch-component challenge

### PR DESCRIPTION
**Pull request by OSAdams**
react-stopwatch-component

### CODE REVIEW

File: react-stopwatch-component/src/stopwatch.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  handleClick() {
+    if (!this.state.isRunning) {
+      this.interval = setInterval(() =>
+        this.setState({ timePassed: this.state.timePassed + 1 }),
+      1000
+      );
+      return this.setState({ isRunning: true });
+    }
+    clearInterval(this.interval);
+    return this.setState({ timePassed: 0, isRunning: false });
```
Comment (thebearingedge) [inline react-stopwatch-component/src/stopwatch.jsx:22]: Pausing the timer should not reset it.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Good work so far, almost there!

**Comment by OSAdams** [comment]
react-stopwatch-component-2

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, review the description again, the user should be able to reset the timer if it's currently paused.

**Comment by thebearingedge** [comment]
By clicking the watch face

**Comment by OSAdams** [comment]
react-stopwatch-component-3

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**